### ISSUE TITLE
Form validation for register page

### ISSUE THREAD

**Issue by bhumika-1-3** [princid/Expense-Tracker#10]

**Comment by chashamsarao** [comment]
Hello, Can I work on the form validation part?

**Comment by princid** [comment]
Yeah sure! I was waiting for someone to help me on this part. Make a pull request after fixing it. 🙌🏻

**Comment by bhumika-1-3** [comment]
I want to work on this

**Comment by princid** [comment]
Sure @bhumika-1-3! Go ahead and make a pull request. I'm assigning you this issue.

**Comment by pssruthy** [comment]
Is anyone working on this issue? I could see the validation part is pending.

<img width="609" alt="Screenshot 2022-10-18 at 10 07 43 PM" src="https://user-images.githubusercontent.com/58027046/196491622-0fabb4b3-4769-424d-9f5e-1fd4134adbab.png">

@princid If no one is working on this could you please assign to me. I would like to work.

**Comment by princid** [comment]
Yeah @pssruthy ! Go ahead, I'm assigning this to you.

**Comment by anil-muraleedharan** [comment]
@princid could you review the PR's raised for this issue, the other [issue](https://github.com/princid/Expense-Tracker/issues/43) I raised for confirm password might have dependency(kind of related or continuation) with this issue. So it would really help if these changes are reviewed and merged.
